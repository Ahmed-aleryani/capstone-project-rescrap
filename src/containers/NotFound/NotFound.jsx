import React from 'react';

const NotFound = (props) => {

  const requestPage = props.match.url;
  // console.log(requestPage);
  return (
    <div className="h-screen flex flex-col justify-center items-center bg-gray-200">
      <h1 className="text-green-100 text-5xl w-11/12 shadow-md px-2 py-2 rounded-lg text-center bg-gray-100">
        Error 404 requested resource "{requestPage}" not found!!! ......... :C{' '}
      </h1>
    </div>
  );
};
NotFound.defaultProps = {

    match:{url:""},
  };

export default NotFound;
