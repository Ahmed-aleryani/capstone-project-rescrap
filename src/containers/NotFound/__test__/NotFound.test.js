import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../NotFound';

it("matches the snapshot", () =>{
    const tree = renderer.create(<NotFound/>).toJSON();
    expect(tree).toMatchSnapshot();
})