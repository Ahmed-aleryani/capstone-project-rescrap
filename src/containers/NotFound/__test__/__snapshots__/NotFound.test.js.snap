// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className="h-screen flex flex-col justify-center items-center bg-gray-200"
>
  <h1
    className="text-green-100 text-5xl w-11/12 shadow-md px-2 py-2 rounded-lg text-center bg-gray-100"
  >
    Error 404 requested resource "
    
    " not found!!! ......... :C
     
  </h1>
</div>
`;
